src/components/EntityStatus/EntityStatus.scss

```diff
@@ -52,6 +44,16 @@
 
         white-space: nowrap;
         text-overflow: ellipsis;
+
+        a {
+            text-decoration: none;
+
+            color: var(--g-color-text-link);
+        }
+
+        a:hover {
+            color: var(--g-color-text-link-hover);
+        }
```

Comment:
Move styles inside `link` class, so link colors won't apply to buttons

Reply:
Why should we define these styles? Is it possible to use proper uikit components with needed styles?

Reply:
React-router `Link` was used, it doesn't have proper styles. Replaced it with out link wrapper and removed unneeded styles